.github/workflows/deploy-frontends-to-production.yml

```diff
@@ -20,23 +20,25 @@ jobs:
             sha=${GITHUB_SHA::8}
             timestamp=$(date +%s)
             datetime=$(date)
-            echo "::set-output name=build_id::prod-${sha}-${timestamp}"
+            echo "::set-output name=build_id::stage-${sha}-${timestamp}"
```

Comment:
@123MwanjeMike could you have done this accidentally.

Reply:
Thanks for pointing these out. I just figured I left in so many errors last night